packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -154,7 +138,12 @@ const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplicati
 });
 
 // Associate application to the attribute group.
-customAttributeGroup.attributeGroup.associateWith(associatedApp.appRegistryApplication());
+associatedApp.appRegistryApplication().addAttributeGroup({
```

Comment:
`appRegistryApplication` should probably be a field, not a function? Make it a getter if you need to throw.